flash/audio/classification/cli.py

```diff
@@ -45,6 +45,7 @@ def audio_classification():
         default_arguments={
             "trainer.max_epochs": 3,
         },
+        legacy=True,
```

Comment:
I assume it's alright to not explain more about this var because it will be phased out shortly after?